gammapy/spectrum/butterfly.py

```diff
@@ -19,13 +19,12 @@ class SpectrumButterfly(QTable):
     - ``flux_hi``
     """
 
-    def plot(self, energy_range=None, ax=None, energy_power=0, **kwargs):
+    def plot(self, energy_range=None, ax=None, energy_power=0,
```

Comment:
The ``SpectrumFitResult`` class has a method ``plot_butterfly``, I think. Can you remove it (since it is superseded by this one?)